### PR TITLE
(node/auxtel-fp01.cp) add el9 network config & tests

### PR DESCRIPTION
**Pull request by dtapiacl**
Convert network configuration from EL7 to EL9 in preparation for its OS upgrade.

### CODE REVIEW

**Review by cbarria — APPROVED** [review]
Looks Great 👍🏻

File: hieradata/node/auxtel-fp01.cp.lsst.org.yaml [hunk 1]
```diff
@@ -1,37 +1,85 @@
...
+        uuid: "17b7e246-327c-76fb-6648-b59ca1bdc81e"
+        type: "ethernet"
+        interface-name: "eno1"
+      ethernet: {}
+      ipv4:
+        method: "auto"
+      ipv6:
+        method: "disabled"
+      proxy: {}
+  eno2:  # dds-auxtel
```
Comment (jhoblitt) [inline hieradata/node/auxtel-fp01.cp.lsst.org.yaml]: I'm unsure if CSCs are still running on auxtel-fp01.  I believe the intent was the would only be running on auxtel-mcm.

This seems OK as is but I would ask if the dds interface configuration can be removed.

**Review by gseriche — APPROVED** [review]
lgtm

File: hieradata/node/auxtel-fp01.cp.lsst.org.yaml (hunk 1 above)
Comment (dtapiacl) [reply hieradata/node/auxtel-fp01.cp.lsst.org.yaml]: According to Tony they have never used it. I will proceed to clean that up.

https://rubin-obs.slack.com/archives/C07QM70QAR2/p1730829490064959

**Review by jhoblitt — CHANGES_REQUESTED** [review]
The spec file name doesn't need to include `.yaml`. E.g. `auxtel-fp01.cp.lsst.org.yaml_spec.rb`

**Comment by dtapiacl** [comment]
> The spec file name doesn't need to include `.yaml`. E.g. `auxtel-fp01.cp.lsst.org.yaml_spec.rb`

You are correct, this was a typo on my end that i did not see. Thank you !

File: hieradata/node/auxtel-fp01.cp.lsst.org.yaml [hunk 2]
```diff
@@ -51,7 +91,7 @@ nfs::client_enabled: true
 nfs::client_mounts:
   /net/self/data:
     share: "data"
-    server: "%{facts.networking.fqdn}"
+    server: "auxtel-fp01.cp.lsst.org"
```
Comment (jhoblitt) [inline hieradata/node/auxtel-fp01.cp.lsst.org.yaml]: Why does this need to be changed?

Comment (dtapiacl) [reply hieradata/node/auxtel-fp01.cp.lsst.org.yaml]: No particular reason, i have updated the code.

**Review by jhoblitt — APPROVED** [review]

**Pull request merged**